packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
+authors: [ "Jon Ludlam" "jonathan.ludlam@citrix.com"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
+license: "LGPL-2.0 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/wsproxy"
+dev-repo: "https://github.com/xapi-project/wsproxy"
```

Comment:
Not sure, but `opam lint` doesn't complain when it ends with `.git`, and that is the format that the other `opam` files use.